### PR TITLE
ansible: fix some bugs in conf and script

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curve-ansible/roles/generate_config/templates/client.conf.j2 [hunk 1]
```diff
@@ -133,6 +133,14 @@ global.logPath={{ client_log_path }}
 # 单元测试情况下
 # logpath=./runlog/
 
+#
+################# read from source conf ###############
+#
+# close fd which read from source file timeout is 300 seconds
+closefd.timeout=300
+# backend thread iterates the fdMap every 600 seconds to close the timeout fd
+closefd.timeInterval=600
```
Comment (wu-hanqing) [inline curve-ansible/roles/generate_config/templates/client.conf.j2:142]: don't write the value of configuration item in here.
refer to other configutation items and write the default value in `curve-ansible/roles/generate_config/defaults/main.yml`.

Comment (SeanHai) [reply curve-ansible/roles/generate_config/templates/client.conf.j2:142]: > don't write the value of configuration item in here.
> refer to other configutation items and write the default value in `curve-ansible/roles/generate_config/defaults/main.yml`.

done

**Review by wu-hanqing — APPROVED** [review]

File: curve-ansible/README.md [hunk 2]
```diff
@@ -374,7 +374,6 @@ ansible-playbook rolling_update_nebd.yml -i client.ini --tags restart
 │   ├── get_curve_version_from_metric.yml                   # 从metric获取curve版本
 │   ├── get_distro_name.yml                                 # 获取系统版本
 │   ├── get_nebd_version_from_metric.yml                    # 从metric获取nebd版本
-│   ├── get_software_version_from_package_version.yml       # 从包版本获取软件版本
```
Comment (wu-hanqing) [inline curve-ansible/README.md:377]: if this file is no longer in use, it can be deleted

Comment (SeanHai) [reply curve-ansible/README.md:377]: > if this file is no longer in use, it can be deleted

done

**Review by wu-hanqing — APPROVED** [review]

**Review by cw123 — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**